.github/workflows/sylius-frankenphp.yml

```
@@ -0,0 +1,241 @@
+name: FrankenPHP Docker Image CI for Sylius
+
+on:
+    workflow_dispatch: ~
+    push:
+        branches: [ main, frankenphp ]
+        paths:
+            - 'frankenphp/**'
+            - 'WORKSPACE'
+    schedule:
+        - cron: '0 20 * * 5'
+
+concurrency:
+    group: sylius_frankenphp_build
+    cancel-in-progress: false
+
+jobs:
+
+    build:
+
+        name: "Sylius FrankenPHP - ${{ matrix.frankenphp }}${{ matrix.php }} ${{ matrix.distro }}"
```

Is  this top level whiteline necessary?

```suggestion
jobs:
    build:
        name: "Sylius FrankenPHP - ${{ matrix.frankenphp }}${{ matrix.php }} ${{ matrix.distro }}"
```